lib/solveEquation/EquationOperations.js

```diff
@@ -224,7 +224,9 @@ function performTermOperationOnExpression(expression, op, term) {
     Node.Creator.parenthesis(expression) : expression);
 
   term.changeGroup = 1;
-  const newNode = Node.Creator.operator(op, [node, term]);
+  const newNode = op === '-'
+    ? Node.Creator.operator('+', [node, Node.Creator.unaryMinus(term)])
+    : Node.Creator.operator(op, [node, term]);
```

Comment:
Since math-parser doesn't output subtract nodes and the new flattenOperands doesn't handle them I thought we may as well handle the operation the way it would be parse.